packages/contracts-sdk/src/implementations/SnickerdoodleWalletContract.ts

```diff
@@ -132,8 +162,9 @@ export class SnickerdoodleWalletContract
 
   public generateAddEVMAddressWithP256KeyChallenge(
     evmAccountAddress: EVMAccountAddress,
-  ): Result<string, InvalidParametersError> {
-    return ok(evmAccountAddress);
+  ): Result<Uint8Array, InvalidParametersError> {
+    // Remove 0x, convert to Uint8Array
```

Comment:
You should check this out, and make sure it works with both 0x123 and just 123. I think that .encode will handle both versions, you should test that.